boilerplate/openshift/golang-osd-operator/prow-config

```diff
@@ -0,0 +1,176 @@
+#!/bin/bash -e
+
+REPO_ROOT=$(git rev-parse --show-toplevel)
+source $REPO_ROOT/boilerplate/_lib/common.sh
+
+cmd=${0##*/}
+
+release_repo=openshift/release
+
+usage() {
+    cat <<EOF
+Usage: $cmd [PATH_TO_RELEASE_CLONE]
+
+Creates a delta in $release_repo standardizing prow configuration for a
+boilerplate consumer. Must be invoked from within a local clone of a repository
+already subscribed to the $CONVENTION_NAME convention.
+
+Parameters:
+    PATH_TO_RELEASE_CLONE   File system path to a local clone of
+                            https://github.com/$release_repo. If not
+                            specified, the repository will be cloned in a
+                            temporary directory.
+EOF
+    exit -1
+}
+
+repo_name() {
+    (git -C $1 config --get remote.upstream.url || git -C $1 config --get remote.origin.url) | sed 's,.*:,,; s/\.git$//'
+}
+
+# Was a release repo clone specified?
+release_clone=
+if [[ $# -eq 1 ]]; then
+    # Special cases for usage queries
+    if [[ "$1" == '-'* ]] || [[ "$1" == help ]]; then
+        usage
+    fi
+
+    [[ -d $1 ]] || err "$1: Not a directory."
+
+    [[ $(repo_name $1) == "$release_repo" ]] || err "$1 is not a clone of $release_repo; or its 'origin' remote is not set properly."
+
+    # Got a usable clone of openshift/release
+    release_clone=$1
+
+elif [[ $# -ne 0 ]]; then
+    usage
+fi
+
+consumer=$(repo_name .)
+[[ -z "$consumer" ]] && err "Failed to determine current repository name"
+consumer_org=${consumer%/*}
+[[ -z "$consumer_org" ]] && err "Failed to determine consumer org"
+consumer_name=${consumer#*/}
+[[ -z "$consumer_name" ]] && err "Failed to determine consumer name"
+# This will be something like refs/remotes/origin/master
+default_branch=$(git symbolic-ref refs/remotes/origin/HEAD)
+[[ -z "$default_branch" ]] && err "Failed to determine default branch name"
+# Strip off refs/remotes/origin/
+default_branch=${default_branch##*/}
+[[ -z "$default_branch" ]] && err "Failed to determine default branch name"
+
+# Make sure we were invoked from a boilerplate consumer.
+[[ -z "$CONVENTION_NAME" ]] && err "$cmd must be invoked from a consumer of an appropriate convention. Where did you get this script from?"
+# Or at least not from boilerplate itself
+[[ "$consumer" == "openshift/boilerplate" ]] && err "$cmd must be invoked from a boilerplate consumer, not from boilerplate itself."
+
+[[ -s $CONVENTION_ROOT/_data/last-boilerplate-commit ]] || err "
+$cmd must be invoked from a boilerplate consumer!"
```

Comment:
Is the newline required?

Reply:
Not required, no; I just found it made things more readable, both in the code and in the output.

But I should really make it consistent throughout :P

Depending on other reviews, I'll fix here or in a fup.

Reply:
+1 to consistancy.